ols/app/endpoints/ols.py

```diff
@@ -103,7 +103,9 @@ def conversation_request(
     # Redact all attachments
     attachments = redact_attachments(conversation_id, attachments)
 
-    # All attachments should be appended to query
+    # All attachments should be appended to query - but store original
+    # query for later use in transcript storage
+    query_without_attachments = llm_request.query
```

Comment:
I guess this is TBD, but I don't really know if we should store the query with attachments AND attachments.

It might be good to have the actual full query to replicate the answers, but it should be possible to construct it anyway (from the `attachments` field in the transcript).

So this is a question of data duplication vs. some use case (?)

Reply:
IMHO we should start first to store + retrieve attachments, not vice versa. IMHO this ^ is not fully solvable